### PR TITLE
LRSD-6615 [CP] Add translations to callback form

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html [hunk 1]
```diff
@@ -13,8 +14,24 @@
 
 [#assign message = "While immediate callbacks are reserved for those with platinum level, a support member will get in touch within 1 business day during our <a href='https://web.liferay.com/support/coverage-areas-and-hours'>standard business hours</a> of your designated Support Center. In the meantime, consider creating a <a href='https://help.liferay.com/hc'>support ticket</a> which can help expedite a resolution." /]
 
+[#if languageId == "es_ES"]
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html:17]: @JoMaAlves can we convert this to a traditional language key in liferay, I am not sure if we can define those in the site initializer or not, but would be cleaner to just use a language key and set it up in the oob language portlet

Comment (JoMaAlves) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html:17]: I'll take a look

Comment (JoMaAlves) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html]: Done!

**Comment by JoMaAlves** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [#assign
 	displayObject = (request.getAttribute("LAYOUT_DISPLAY_PAGE_OBJECT_PROVIDER").getDisplayObject())!{}
 	objectEntryJSON = jsonFactoryUtil.createJSONObject()
+	languageId = themeDisplay.getLanguageId()
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html:4]: @JoMaAlves I dont think this line is needed anymore, but the rest of the changes look great!

Comment (JoMaAlves) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html:4]: Oops, my bad

Comment (JoMaAlves) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html]: Done!

**Comment by ryanschuhler** [comment]
ci:forward

**Comment by ryanschuhler** [comment]
ci:forward:force

**Comment by ryanschuhler** [comment]
ci:forward:force

**Pull request closed without merging**